Standup-Meetings–Reports.md

```diff
@@ -0,0 +1,54 @@
+ # Standup Meetings – Reports
+
+The main purpose of scrum reports is to collect and analyze data along the way to accomplishing the right project, at the right time using minimum iterations.
```

Comment:
I think we should remove this first section from "Reports" section and start directly from "After the stand-up in concluded.." section

Reply:
Okay. I will remove the first few paragraphs and resume from "After the stand-up concluded", as suggested.